pennylane/templates/tensornetworks/mps.py

```diff
@@ -46,38 +47,47 @@ def compute_indices_MPS(wires, n_block_wires):
             f"n_block_wires must be smaller than or equal to the number of wires; got n_block_wires = {n_block_wires} and number of wires = {n_wires}"
         )
 
-    if n_wires % (n_block_wires / 2) > 0:
+    if not abs(offset) < n_block_wires / 2:
+        raise ValueError(
+            f"Absolute value of offest must be smaller than half of the n_block_wires; got offset = {offset} and n_block_wires = {n_block_wires}"
+        )
```

Comment:
I have included the support for odd-numbered-wires block, but I would still want to stick to the offset w.r.t. to the wires so that it is easier to correspond to the literature. 

🥊 🥊